### PR TITLE
Add option --compile=min to Julia man page.

### PR DESCRIPTION
**Pull request by derekmahar**
Add option `--compile=min` to Julia man page that is present in `julia --help-hidden`.

### CODE REVIEW

**Comment by derekmahar** [comment]
I'd like to change the commit comment to "Add option --compile=min to Julia man page.".

File: doc/man/julia.1 [hunk 1]
```diff
@@ -130,7 +130,7 @@ Enable or disable color text
 Load or save history
 
 .TP
---compile={yes|no|all}
+--compile={yes|no|all|min}
 Enable or disable compiler, or request exhaustive compilation
```
Comment (StefanKarpinski) [inline doc/man/julia.1:134]: The description probably needs an update as well.

Comment (derekmahar) [reply doc/man/julia.1:134]: Let me make a new commit and pull request.

Comment (StefanKarpinski) [reply doc/man/julia.1:134]: You can just push more commits here and it can be squashed upon merging.

Comment (mbauman) [reply doc/man/julia.1:134]: It looks like you did this with the GitHub web interface — you can just keep editing here:

https://github.com/derekmahar/julia/edit/patch-1/doc/man/julia.1?pr=%2FJuliaLang%2Fjulia%2Fpull%2F36585

(Welcome, by the way!)

**Comment by derekmahar** [comment]
Please ignore this PR.

**Pull request closed without merging**